packages/salesforce-adapter/src/filters/custom_objects_instances.ts

```diff
@@ -531,6 +554,14 @@ const createInvalidManagedBySaltoFieldFetchWarning = async (
   severity: 'Warning',
 })
 
+const createInaccessibleFieldsFetchWarning = async (
+  objectType: ObjectType,
+  inaccessibleFields: string[],
+): Promise<SaltoError> => ({
+  message: `The following fields are not accessible and will not be fetched for records of type ${await apiName(objectType)}: ${inaccessibleFields.join(',')}. Values of these fields will appear as 'deleted' if these data records are ever deployed`,
```

Comment:
```suggestion
  message: `The following fields are not accessible and will not be fetched for records of type ${apiNameSync(objectType)}: ${inaccessibleFields.join(',')}. Values of these fields will appear as 'deleted' if these data records are ever deployed`,
```

We can make this sync now 🥳